src/ztc/api/serializers/besluittype.py

```diff
@@ -1,32 +1,23 @@
-from rest_framework_nested.relations import NestedHyperlinkedRelatedField
-from rest_framework_nested.serializers import NestedHyperlinkedModelSerializer
+from rest_framework import serializers
 
-from ...datamodel.models import BesluitType
+from ...datamodel.models import BesluitType, InformatieObjectType
 
 
-class BesluitTypeSerializer(NestedHyperlinkedModelSerializer):
-    parent_lookup_kwargs = {
-        'catalogus_uuid': 'catalogus__uuid'
-    }
-
-    informatieobjecttypes = NestedHyperlinkedRelatedField(
+class BesluitTypeSerializer(serializers.HyperlinkedModelSerializer):
+    informatieobjecttypes = serializers.HyperlinkedRelatedField(
+        source='catalogus.informatieobjecttype_set',
```

Comment:
should be `informatieobjecttypes` - this is a M2M between `BesluitType` and `InformatieObjectType`. In its current form I think this will list _all_ IOT in the catalogus instead of just those connected to the `BesluitType`.
```suggestion
```